crates/storage/src/schema/revision_0038.rs

```diff
@@ -2,13 +2,19 @@ use anyhow::Context;
 
 /// This migration removes the event key FTS5 table for v0.2 RPC.
 pub(crate) fn migrate(tx: &rusqlite::Transaction<'_>) -> anyhow::Result<()> {
-    tx.execute_batch(
-        "
-DROP TRIGGER starknet_events_ad;
-DROP TRIGGER starknet_events_ai;
-DROP TRIGGER starknet_events_au;
-DROP TABLE  starknet_events_keys;
-    ",
-    )
-    .context("Dropping v0.2 events FTS5 data and triggers")
+    tracing::info!("Removing event event data for RPC v0.2, this may take some time.");
+    tx.execute("DROP TRIGGER starknet_events_ad", [])
+        .context("Dropping trigger starknet_events_ad")?;
+    tracing::info!("Dropped trigger 1 of 3, 25% complete");
+    tx.execute("DROP TRIGGER starknet_events_ai", [])
+        .context("Dropping trigger starknet_events_ai")?;
+    tracing::info!("Dropped trigger 2 of 3, 50% complete");
+    tx.execute("DROP TRIGGER starknet_events_au", [])
+        .context("Dropping trigger starknet_events_au")?;
+    tracing::info!("Dropped trigger 3 of 3, 75% complete");
+    tx.execute("DROP TABLE starknet_events_keys", [])
+        .context("Dropping table starknet_events_keys")?;
+    tracing::info!("Dropped events table, 100% complete");
```

Comment:
These are quite misleading I guess. Removing the triggers is pretty much instant -- it's just the `DROP TABLE` that takes some time...

Reply:
Personally I'd just leave the code as it was (using `execute_batch()`) and just keep the `info` message.